script.js

```diff
@@ -1,4 +1,159 @@
+    playerTwoDiceTwo = rollDices();
+    playerTwoDiceRoll = "rolled";
+    return `Welcome Player 2! <br><br>
+    You rolled...<br>
+    # Dice One: ${playerTwoDiceOne}<br>
+    # Dice Two: ${playerTwoDiceTwo} <br><br>
+    Please type in either '1' or '2' to continue... <br>
+    # Option 1: ${playerTwoDiceOne}${playerTwoDiceTwo} <br>
+    # Option 2: ${playerTwoDiceTwo}${playerTwoDiceOne}`;
+  }
+
+  // if player two has rolled the dice, ask player two to choose arrangement
+
+  // if player two chooses option 1 (arrangement of dice 1+2)
+  if (playerTwoDiceRoll == "rolled" && playerTwoChoice == "" && input == "1") {
+    playerTwoChoice = Number(
+      String(playerTwoDiceOne) + String(playerTwoDiceTwo)
+    );
+
+    // if player two loses
+    if (playerTwoChoice < playerOneChoice) {
+      return `Player 2, your choice is ${playerTwoChoice}. <br>
+      Player 1's choice is ${playerOneChoice}. <br><br>
+      Player 1 wins, Player 2 loses!`;
+    }
+
+    // if player two wins
+    if (playerTwoChoice > playerOneChoice) {
+      return `Player 2, your choice is ${playerTwoChoice}. <br>
+      Player 1's choice is ${playerOneChoice}. <br><br>
+      Player 1 loses, Player 2 wins!`;
+    }
+
+    // if draw
+    if (playerTwoChoice == playerOneChoice) {
+      return `Player 2, your choice is ${playerTwoChoice}. <br>
+      Player 1's choice is ${playerOneChoice}. <br><br>
+      It's a draw!`;
+    }
+  }
+
+  // if player two chooses option 2 (arrangement of dice 2+1)
+  if (playerTwoDiceRoll == "rolled" && playerTwoChoice == "" && input == "2") {
+    playerTwoChoice = Number(
+      String(playerTwoDiceTwo) + String(playerTwoDiceOne)
+    );
+
+    // if player two loses
+    if (playerTwoChoice < playerOneChoice) {
+      return `Player 2, your choice is ${playerTwoChoice}. <br>
+      Player 1's choice is ${playerOneChoice}. <br><br>
+      Player 1 wins, Player 2 loses!`;
+    }
+
+    // if player two wins
+    if (playerTwoChoice > playerOneChoice) {
+      return `Player 2, your choice is ${playerTwoChoice}. <br>
+     Player 1's choice is ${playerOneChoice}. <br><br>
+     Player 1 loses, Player 2 wins!`;
+    }
+
+    // if draw
+    if (playerTwoChoice == playerOneChoice) {
+      return `Player 2, your choice is ${playerTwoChoice}. <br>
+    Player 1's choice is ${playerOneChoice}. <br><br>
+    It's a draw!`;
+    }
+  }
+
+  // if player two does not choose any of the options
+  if (playerTwoChoice == "") {
+    return `Player 2, please type in either '1' or '2' to continue... <br><br>
+      # Option 1: ${playerTwoDiceOne}${playerTwoDiceTwo} <br>
+      # Option 2: ${playerTwoDiceTwo}${playerTwoDiceOne}`;
+  }
+};
+
+var rollDices = function () {
+  // Generate a decimal inclusive of 0 and exclusive of 6
+  var randomDecimal = Math.random() * 6;
+  // Remove the decimal with the floor operation to get an integer from 1 to 6 inclusive
+  var randomInteger = Math.floor(randomDecimal) + 1;
+  return randomInteger;
 };
```

Comment:
this was shown step by step in the preclass to simplify and make it easier to understand but consider condensing line 15-18 into one line

return Math.floor(Math.random() * 6) + 1